exercise-1/script.js

```diff
@@ -0,0 +1,22 @@
+const button = document.querySelector('button');
+const nav = document.querySelector('nav');
+const main = document.querySelector('.hide-main');
+
+button.addEventListener('click', function () {
+  nav.classList.toggle('display');
+  main.classList.toggle('hide-main');
+});
+
+function keyPress(e) {
+  if (e.key === 'Escape') {
+    console.log('hello');
+  } else false;
+}
+
+document.onkeydown = function (evt) {
+  evt = evt || window.event;
+  if (evt.keyCode == 27) {
```

Comment:
`event.keyCode` is deprecated ([see here](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)). Use `event.key` instead.